pytest.ini

```diff
@@ -3,6 +3,7 @@ testpaths = tests
 markers =
     develop: Test in develop
     smoke: Smoke tests
+    no_protocol: Protocol-independent tests
```

Comment:
Maybe we could name it no_connection or connectionless. What do you think?

Reply:
Done!